.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,6 +14,7 @@ Fixes #
 - [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
 - [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
 - [ ] Write detailed docstrings for all functions/methods.
+- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
 - [ ] If adding new functionality, add an example to docstrings or tutorials.
```

Comment:
I feel like these checklist items could be consolidated and/or reworded to be a bit better. Any suggestions?

Reply:
I agree. Perhaps the following two could be consolidated to "Try to get all checks passing, including style checks and test coverage."

>  - Run make format and make check to make sure the code follows the style guide.
> - Add tests for new features or tests that would have caught the bug that you're fixing.

For the following four, if you want I could add a section to 'Contributing code' for adding new functions/methods/classes (including GMT wrappers) that covers this information such that the checklist is simply 'Follow guide in `<link to contributing guide section>` if adding a new function/method/class'.

> - Add new public functions/methods/classes to doc/api/index.rst.
> - Write detailed docstrings for all functions/methods.
> - If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
> - If adding new functionality, add an example to docstrings or tutorials.

Reply:
@weiji14, since this PR has a few approvals already would you mind if I merged this and worked on your suggestions in a separate PR? That way the other reviewers would not need to re-look at existing work. I have the updates partly done, but realistically won't have time to finish it until at least late next week due to AGU rush, GMT tasks, and planned leave.

Reply:
Ok, sounds good to me to merge this new issue template first. The Pull Request template does need a bit of a refresh so that can be done in a follow up Pull Request (but please do prioritize AGU first).